### ISSUE TITLE
Can't seem to generate a splitter

### ISSUE THREAD

**Issue by AlicePain** [kendo-labs/knockout-kendo#199]
Have a JSFiddle to help with this; same issue in my application.  I don't know why this isn't working it seems fine on your site.
https://jsfiddle.net/tptm75cy/

No splitter shows, just the Pane One/Pane Two div

**Comment by rniemeyer** [comment]
Hello- Some of the resource links in your fiddle were not working. Here is a working copy: https://jsfiddle.net/rniemeyer/c8xubx35/ . Maybe you can compare it to your code. Also, the order of the scripts is important jQuery, Kendo, Knockout, and then Knockout-Kendo (Kendo and Knockout could be interchanged). Happy to try to help further

**Comment by AlicePain** [comment]
It does indeed work in JSFiddle, thanks for the help

**Comment by AlicePain** [comment]
I have a large SPA with Knockout/RequireJS and Kendo, my other kendo controls work just fine.  

For some reason the splitter just wouldn't render no matter what I did.  Even a trigger(resize) didn't work.  Finally I realized the splitter was valid but trying $("#splitter").data("kendoSplitter") was always undefined.  So I added the following: $("#splitter").kendoSplitter(); and it rendered.

Thank you for the JSFiddle help, that was my first time trying to create one and knowing that you got it working kept me going to resolve it in my app.
